app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -97,46 +99,53 @@ living costs, to train (depending on your immigration status, you may get a redu
 
 ## Apply for teacher training
 
-We can help you make an application for teacher training in England. There are a number of steps to follow.
+We can help you make an application for teacher training in England. Sign up for a teacher training adviser to get help right away and throughout the application process.
+
+### 1. Get support and advice
+
+For help applying, you can:
 
-### 1. Check your qualifications
+* sign up to [get a teacher training adviser](https://adviser-getintoteaching.education.gov.uk/) – you’ll be asked for your phone number, email and location so a dedicated adviser can call you back in your time zone
+* call +44 800 389 2500, Monday to Friday between 8:30am and 5:30pm (UK local time)  – calls will be charged at your standard rate
+* use the live chat service at the bottom of this page
 
-To teach children aged 11 to 18 in secondary school, you’ll need:
 
-- a bachelor’s degree (this can be from a university outside the UK)
-- to meet a standard in English and maths equivalent to a grade 4 GCSE (General Certificate of Secondary Education)
+### 2. Check your eligibility
 
-To teach children aged 3 to 11 in primary school, you will also need to meet a standard in physics, chemistry or biology equivalent to a grade 4 GCSE.
+To train to teach in England, you will need the following qualifications:
+
+* a bachelor’s degree (this can be from a university outside the UK)
+* an English language qualification which is the same standard as a grade 4 GCSE (General Certificate of Secondary Education)
+* a maths qualification which is the same standard as a grade 4 GCSE
+* to teach children aged 3 to 11 in primary school, a physics, chemistry or biology (science) qualification which is the same standard as a grade 4 GCSE
```

Comment:
```suggestion
* a physics, chemistry or biology (science) qualification which is the same standard as a grade 4 GCSE (to teach children aged 3 to 11 in primary school )
```
not sure if the current line quite works with the lead in line - how about this?

Reply:
I think it's OK with the lead in line and I don't want users to think this is a requirement for all teacher training, which they might do if they don't get to the end of the sentence...

Reply:
I agree about this bullet not quite working with the lead in line. How about amending slightly to:
* if teaching children aged 3 to 11 in primary school, a physics, chemistry or biology (science) qualification which is the same standard as a grade 4 GCSE

Reply:
That looks fine, thanks @GDW7